watertap/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -372,14 +364,36 @@ def solve(blk, solver=None, tee=False, check_termination=True):
     return results
 
 
-def initialize_system(m, solver=None):
+def initialize_system(m, solver=None, relaxed_initialization=False):
     if solver is None:
         solver = get_solver()
     optarg = solver.options
 
     # ---initialize RO---
-    m.fs.RO.initialize(optarg=optarg)
+    if relaxed_initialization:
+        print("Initializing RO with relaxed recovery specification")
+        m.fs.RO.recovery_mass_phase_comp[0, "Liq", "H2O"].unfix()
+        target_recovery = m.fs.RO.recovery_mass_phase_comp[0, "Liq", "H2O"].value
+        m.fs.RO.area_objective = Objective(
+            expr=(m.fs.RO.recovery_mass_phase_comp[0, "Liq", "H2O"] - target_recovery)
+            ** 2
+        )
 
+        expected_DOFs = 1
+        print(
+            f"Unfixed membrane area, and added objective to achieve target recovery of {target_recovery}"
+        )
+        m.fs.RO.initialize(
+            optarg=optarg, initialization_degrees_of_freedom=expected_DOFs
+        )
+        m.fs.RO.recovery_mass_phase_comp[0, "Liq", "H2O"].fix(target_recovery)
+        m.fs.RO.area_objective.deactivate()
```

Comment:
We should probably just delete the objective instead of deactivating it, and check that the deletion is successful.